CMakeLists.txt

```diff
@@ -44,38 +50,59 @@ configure_file(
     @ONLY
 )
 
-set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
Would prefer not to remove this, can you explain why this is relevant?

Reply:
Maybe because it might affect compile arguments, violating [CMAKE.PASSIVE_PROJECTS]

Reply:
This determines the CXX standard that CMake will use **[CMAKE.PASSIVE_PROJECTS]** states that:

> User-provided compilation flags, whether specified via presets, command-line options, or toolchains, must not be modified by CMake projects. Therefore, CMake projects may not set variables that impact compilation flags

If you want to build and test with extensions turned off, do so, but if the user wants extensions turned on globally via their toolchain file or whatever a project shouldn't modify that default.

Reply:
Fixed